include/SFML/Graphics/Font.hpp

```diff
@@ -348,14 +349,14 @@ class SFML_GRAPHICS_API Font
     ////////////////////////////////////////////////////////////
     void*                      m_library;     ///< Pointer to the internal library interface (it is typeless to avoid exposing implementation details)
     void*                      m_face;        ///< Pointer to the internal font face (it is typeless to avoid exposing implementation details)
-    void*                      m_streamRec;   ///< Pointer to the stream rec instance (it is typeless to avoid exposing implementation details)
+    std::shared_ptr<void>      m_streamRec;   ///< Pointer to the stream rec instance (it is typeless to avoid exposing implementation details)
     void*                      m_stroker;     ///< Pointer to the stroker (it is typeless to avoid exposing implementation details)
-    int*                       m_refCount;    ///< Reference counter used by implicit sharing
+    std::shared_ptr<int>       m_refCount;    ///< Reference counter used by implicit sharing
```

Comment:
I have no idea why this needs to be a shared_ptr

Reply:
According to https://github.com/SFML/SFML/blob/master/src/SFML/Graphics/Font.cpp it was already a pointer so that it can be shared by several Font instances.

The shared_ptr makes it easy to implement the copy constructor of sf::Font and still share a ref to the same int.